R/Prediction.R

```diff
@@ -138,7 +138,16 @@ makePrediction.SurvTaskDesc = function(task.desc, row.names, id, truth, predict.
   # FIXME: recode times
   data$truth.time = truth[, 1L]
   data$truth.event = truth[, 2L]
-  data$response = y
+  if (predict.type == "response") {
+    data$response = y
+  } else {
```

Comment:
Can you add an explicit check for whether the predict type is probs here please?

Reply:
I think this is not necessary, as predict.type can only be response or prob. If you try to makeLearner with other character it will give you an error.